docs/sources/panels-visualizations/query-transform-data/expression-queries/index.md

```diff
@@ -238,6 +238,157 @@ Resample changes the time stamps in each time series to have a consistent time i
   - **backfill** with next known value
   - **fillna** to fill empty sample windows with NaNs
 
+#### SQL
+
+The SQL expression engine is a new expression engine that allows you to transform the data returned from your data source queries using a MySQL-like syntax.
+
+##### SQL Functions
+
+A subset of MySQL functions are supported in expressions, for two reasons:
+
+1. We use [go-mysql-server](https://github.com/dolthub/go-mysql-server) to execute the SQL statements. It supports a subset of MySQL functions, as per the documentation [here](https://docs.dolthub.com/sql-reference/sql-support/expressions-functions-operators#functions-and-operators).
+2. We restrict the use of potentially dangerous functions.
```

Comment:
Should we include a call-to-action? Something like:
"Open an issue on the Grafana repo if you'd like a function added to this list"

In the Dolt docs they say this ([here](https://docs.dolthub.com/sql-reference/sql-support#:~:text=Any%20deviation%20from%20the%20MySQL%20manual%20should%20be%20documented%20on%20this%20page%2C%20or%20else%20indicates%20a%20bug.%20Please%20file%20issues%20with%20any%20incompatibilities%20you%20discover.)):
> Any deviation from the MySQL manual should be documented on this page, or else indicates a bug. Please [file issues](https://github.com/dolthub/dolt/issues) with any incompatibilities you discover.